docs/index.tpl.html

```diff
@@ -407,11 +406,22 @@ <h1>CSS Variables</h1>
 				<td><code>--mv-item-height</code></td>
 				<td><a href="/docs/collections/#collectiion-editing-controls">Item bar</a> of <a href="/docs/collections/#simple-collections">simple collections</a></td>
 				<td>Height of item in px, e.g. <code>100px</code>.</td>
-			</tr>
+			</tr> -->
 		</tbody>
 	</table>
+
+
 </section>
 
+<div hidden>
+	<select property="live" id="live">
+		<option value="✘">✘</option>
+		<option value="✓">✓</option>
```

Comment:
Since we now have a select, we can store different values than those presented. ✓ and ✘ are more presentational. E.g. what if in the future we want to change them to 👍 and 👎? I'd suggest using them as the visible text, but using values like "yes" and "no" (or whatever your preference is, if different!)
That way, your expressions will also be easier to write, as you'll be comparing with "yes" and "no" instead of weird unicode characters :)

In fact, ✘ can remain empty (`value=""`), and we can keep the CSS we had for adding it. That would reduce the data too, since empty values are not stored.